rel/runtime_config.exs

```diff
@@ -0,0 +1,15 @@
+use Mix.Config
+
+[
+  "PRODUCTION_ENVIRONMENT"
+]
+|> Enum.each(fn config_key ->
+  if System.get_env(config_key) == nil do
+    raise "Config not set in environment: #{config_key}"
+  end
+  config :belfrage_ccp,
+         Keyword.new([{String.to_atom(String.downcase(config_key)), System.get_env(config_key)}])
+end)
```

Comment:
Same question about this as asked on the other PR.

Reply:
Its setting the `PRODUCTION_ENVIRONMENT` variable in the list, I just copied what we do there from belfrage/ccp